integration/aggregate_stats_compat_test.go

```diff
@@ -132,13 +134,17 @@ func TestAggregateCompatCollStats(t *testing.T) {
 				})
 			}
 
+			if setup.IsSQLite(tt) {
```

Comment:
Does this line need a `TODO` too? When https://github.com/FerretDB/FerretDB/issues/3148 is implemented, we should stop exiting here, right?

Reply:
Yeah we can actually add it